packages/smooth_app/lib/pages/product/product_image_viewer.dart

```diff
@@ -126,7 +126,41 @@ class _ProductImageViewerState extends State<ProductImageViewer> {
                   child: Padding(
                     padding:
                         const EdgeInsets.symmetric(horizontal: SMALL_SPACE),
-                    child: _getUnselectImageButton(appLocalizations),
+                    child: (imageProvider == null)
```

Comment:
This is impossible.

Reply:
Sorry, didn't notice that. fixing that asap!